rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -464,6 +464,7 @@ maas_pip_packages:
   - rackspace-monitoring-cli
   - ipaddr
   - swift
+  - waxeye
```

Comment:
Was this a new AIO? The repo server should build this package because its included in this list.